login/src/main/resources/templates/web/change_password.html

```diff
@@ -4,11 +4,28 @@
       layout:decorator="layouts/main">
 <head>
     <th:block layout:include="nav :: head"></th:block>
+    <th:block layout:include="pw_validation :: head"></th:block>
 </head>
 <div class="header" layout:fragment="page-header">
     <th:block layout:include="nav :: nav"></th:block>
 </div>
 <div class="island" layout:fragment="page-content">
+  <script th:inline="javascript">
```

Comment:
This may not be possible in this system, but this code shoudln't be duplicated. Like it should be imported from one location. If this is hard to do within this java system though, I'm fine leaving it and having a TODO note.